File: mobildiafonsdk/src/main/java/com/mobildiafon/rtc/Apartment.java
package com.mobildiafon.rtc;

/** DiafonBox için basit daire modeli (box-activate'ten gelir). */
public final class Apartment {
    public final String apartmentId;
    public final String flatNo;
    public final String name;   // opsiyonel sakin/daire adı
    public Apartment(String apartmentId, String flatNo, String name) {
        this.apartmentId = apartmentId;
        this.flatNo = flatNo;
        this.name = name == null ? "" : name;
    }
    @Override public String toString() { return "Daire " + flatNo; }
}

File: mobildiafonsdk/src/main/java/com/mobildiafon/box/Apartment.kt
package com.mobildiafon.box

/**
 * DiafonBox'ın aradığı daire. box-activate cevabından gelir.
 *  - apartmentId : call:start-flat'e gönderilen kimlik
 *  - flatNo      : ekranda gösterilecek daire no ("20")
 *  - name        : varsa etiket/sakin adı (opsiyonel)
 */
data class Apartment(
    val apartmentId: String,
    val flatNo: String,
    val name: String = ""
)
